src/azure-cli/azure/cli/command_modules/appservice/logicapp/_help.py

```diff
@@ -0,0 +1,53 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.help_files import helps
+
+# pylint: disable=line-too-long
+
+helps['logicapp'] = """
+type: group
+short-summary: Manage logic apps.
+"""
+
+helps['logicapp create'] = """
+type: command
+short-summary: Create a logic app.
+long-summary: The logic app's name must be able to produce a unique FQDN as AppName.azurewebsites.net.
+examples:
+  - name: Create a basic logic app.
+    text: >
+        az logicapp create -g MyResourceGroup --subscription MySubscription -p MyPlan -n MyUniqueAppName -s MyStorageAccount
+"""
+
+helps['logicapp delete'] = """
+type: command
+short-summary: Delete a logic app.
+examples:
+  - name: Delete a logic app. (autogenerated)
+    text: az logicapp delete --name MyLogicapp --resource-group MyResourceGroup --subscription MySubscription
+    crafted: true
```

Comment:
Is this code automatically generated by CodeGen? If not, please delete this tag ` crafted: true`

Reply:
Not sure. I didn't generate the message using CodeGen but a lot of other help texts in appservice/_help.py seem to have that tag.

@calvinsID @panchagnula can you please confirm?

Reply:
Yeah you can just remove!

Reply:
Removed tag.